### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by karammi**
please check my REDME file , it's had some of libs that we want to work on them

### CODE REVIEW

**Review by helloseyedjafari — CHANGES_REQUESTED** [review]
afarin karet khobe damet garm

lotfan esme file ro hamon README.md 
bezar

**Comment by helloseyedjafari** [comment]
lotfan be dev PR bede

**Review by aliraghebi — CHANGES_REQUESTED** [review]
It has some grammatical mistake

File: README_INTRO.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# StarPlayer
+# A player to learn from!  #
+
+**This is a open source music player applciation for learning many of libraries such as coroutines, RxJava, Dagger and ...**
```
Comment (aliraghebi) [inline README_INTRO.md]: This is AN open ...

File: README_INTRO.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+We will be working on it with to learn new things as has been introduced as a good things in android worlds.
```
Comment (aliraghebi) [inline README_INTRO.md]: .... on it to learn ....

.... as good things ....

.... in android world ....

File: README_INTRO.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+<kbd>⌘L</kbd>
+It will involve :  
+   * Architecture: **MVVM**    
+   * Reactive Programming: **RxJava , Coroutines**  
+   * Dependency Injector: **Dagger**  
+   * Network Library: **Retrofit**  
+   * Android Architecture Components: **LiveData,ViewModels, Paging, WorkManager**  
+   **Epoxy**  
+   * ...
```
Comment (hosseiniSeyRo) [inline README_INTRO.md:17]: add this line 
* exoplayer

**Review by hosseiniSeyRo — CHANGES_REQUESTED** [review]

**Review by aliraghebi — COMMENTED** [review]
Comments added

**Review by helloseyedjafari — APPROVED** [review]
good to get started

**Review by aliraghebi — APPROVED** [review]

**Pull request closed without merging**